test/resources/Connections.spec.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+var telnyx = require('../../testUtils').getTelnyxMock();
+var expect = require('chai').expect;
+
+var TEST_AUTH_KEY = 'KEY187557EC22404DB39975C43ACE661A58_9QdDI7XD5bvyahtaWx1YQo';
```

Comment:
All looks good, though at some point in the future we should probably see about removing the secret somehow

Reply:
yeah this secret is just a made up one anyway